docs/pages/environment_creation/index.md

```diff
@@ -5,116 +5,329 @@ title: Environment Creation
 # How to create new environments for Gym
 
 This documentation overviews creating new environments and relevant useful wrappers, utilities and tests included in OpenAI Gym designed for the creation of new environments.
+You can clone gym-examples to play with the code that are presented here:
 
-## Example Custom Environment
+```console
+git clone https://github.com/Farama-Foundation/gym-examples
+pip install -e gym-examples
+```
+
+## Subclassing gym.Env
+
+Before learning how to create your own environment you should check out [the documentation of Gym's API](https://www.gymlibrary.ml/pages/api/index).
 
-Here is a simple skeleton of the repository structure for a Python Package containing a custom environment. For a more complete example, please refer to: https://github.com/openai/gym-soccer.
+We will be concerned with a subset of gym-examples that looks like this:
 
 ```sh
-gym-foo/
+gym-examples/
   README.md
   setup.py
-  gym_foo/
+  gym_examples/
     __init__.py
     envs/
       __init__.py
-      foo_env.py
-      foo_extrahard_env.py
+      grid_world.py
 ```
 
-## Subclassing gym.Env
+For a more complete example, please refer to: https://github.com/openai/gym-soccer.
```

Comment:
I don't think referring to this will be helpful, it's an ancient and unsupported repo

Reply:
:+1: